core/src/test/scala/integration/kafka/api/EndToEndAuthorizationTest.scala

```diff
@@ -133,7 +133,8 @@ trait EndToEndAuthorizationTest extends IntegrationTestHarness with SaslSetup {
   this.serverConfig.setProperty(KafkaConfig.AuthorizerClassNameProp, classOf[SimpleAclAuthorizer].getName)
   // Some needed configuration for brokers, producers, and consumers
   this.serverConfig.setProperty(KafkaConfig.OffsetsTopicPartitionsProp, "1")
-  this.serverConfig.setProperty(KafkaConfig.OffsetsTopicReplicationFactorProp, "1")
+  this.serverConfig.setProperty(KafkaConfig.OffsetsTopicReplicationFactorProp, "3")
+  this.serverConfig.setProperty(KafkaConfig.OffsetCommitRequiredAcksProp, "1")
```

Comment:
you mean, if OffsetsTopicReplicationFactorProp=3. 
I could not figure out why :(